README.md

```diff
@@ -1,36 +1,62 @@
 # Tic Tac Toe Ruby Game (The Odin Project)
 
-> This project is part of the Microverse Ruby Curriculum and The Oding Project. 
+> This is a Tic Tac Toe game made with Ruby. It's a part of the Microverse Ruby Curriculum and The Odin Project. 
 
-## Built With
+## Built using
 
 - Ruby
+- Gitflow Workflow
+
+## UML Class diagram
+
+![screenshot](./UML_Class_diagram.png)
+
+## Project requirements
+- All user interaction and game flow should be kept only in main.rb
+- Use OOP
 
 ## Live Demo
 [Live preview:](https://repl.it/join/gjrzjjbm-marcoshdezcam1)
```

Comment:
- [ ] **MANDATORY** Please add instructions if you are going to include the live demo in repl.it. It seems that repl.it needs some extra configuration 
![image](https://user-images.githubusercontent.com/53256139/91685075-dc8d1080-eb1e-11ea-82c8-97c8031e8f7d.png)
- [ ] **OPTIONAL** You can use gitpod to run complex projects, just append `gitpod.io/` before the link to your repo and branch: https://gitpod.io/github.com/marcoshdezcam/Tic_Tac_Toe_Game/tree/feature_readme_game_instructions